production/helm/loki/templates/ingester/statefulset-ingester.yaml

```diff
@@ -87,7 +87,6 @@ spec:
           {{- end }}
           args:
             - -config.file=/etc/loki/config/config.yaml
```

Comment:
```suggestion
            - -config.file=/etc/loki/config/config.yaml
            - -ingester.availability-zone=zone-default
```

Reply:
I tested this parameter in my live manifest and it works, if it is set to "-ingester.availability-zone=zone-default", my ingester pods are running fine now.